pkgs/by-name/dn/dnsmap/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dnsmap";
+  tag = "0.36";
+  version = "${finalAttrs.tag}-unstable-2024-08-08";
+
+  src = fetchFromGitHub {
+    owner = "resurrecting-open-source-projects";
+    repo = "dnsmap";
+    rev = "b5b4b1a7764e141f2551584d9fad3a973951eafe";
+    hash = "sha256-RTZe0kb/Fq9kIdnHQO//OP+Uop2Z5r22Z7+rQdCFsl4=";
+  };
+
+  preConfigure = ''
+    autoreconf -i
+  '';
+
+  configureFlags = [ "--prefix=$(out)" ];
+
+  buildInputs = [
+    autoconf
+    automake
```

Comment:
Enable:
> strictDeps = true;

Good odds this is going to `nativeBuildInputs`.
Test cross compiling with `pkgsCross`.

> nix build .#pkgsCross.aarch64-multiplatform.dnsmap